.ruby-gemset

```diff
@@ -0,0 +1 @@
+heartcheck
```

Comment:
bundle works fine, but people develop in many forms (rvm, rbenv, etc), what you think about disable this behaviour on your machine?

Like this (are you using rvm?) :
```echo "export rvm_ignore_gemsets_flag=1" >> ~/.rvmrc```